.github/workflows/build.yml

```diff
@@ -114,7 +114,7 @@ jobs:
 
       - uses: actions/setup-dotnet@v1
         with:
-          dotnet-version: "3.1.201"
+          dotnet-version: "6.0.*"
```

Comment:
I guess it should be `6.0.x` when looking the [example]. 🙏🏻 

[example]: https://github.com/actions/setup-dotnet

Reply:
ok I got it. thx 👍

Reply:
`6.0.*` seems to work too 🤔  https://github.com/planetarium/NineChronicles.Headless/blob/c8ff0427cbf3d1b547eb63fc2331841eece94ef7/.github/workflows/lint.yml#L13

Reply:
Oops, the both seems to work. Sorry for my mistake. 😞

Reply:
I think it doesn't matter it's the same result anyway. Thank you, everyone!